src/Altinn.AccessManagement/Controllers/DelegationsController.cs

```diff
@@ -301,5 +301,48 @@ public async Task<ActionResult<List<DelegationExternal>>> GetAlInboundDelegation
                 return StatusCode(500);
             }
         }
+
+        /// <summary>
+        /// Endpoint for retrieving delegated resources between parties
+        /// </summary>
+        /// <response code="400">Bad Request</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpGet]
+        [Route("accessmanagement/api/v1/admin/delegations/maskinportenschema")]
+        [Authorize]
+        public async Task<ActionResult<List<MPDelegationExternal>>> GetAllDelegationsForAdmin([FromQuery] int? supplierOrg, int? consumerOrg, string scope)
+        {
+            if (supplierOrg == null || supplierOrg == 0)
```

Comment:
Why are the properties nullable input if they should not be nullable? Nullables can also be checked by .HasValue and have the value on .Value (instead of casting later).

Also have you checked the Altinn 2 maskinporten API whether any of these are optional there?

Reply:
In the existing, it is scope that is mandatory. I have now modified to reflect the same